### PR TITLE
feat(core/xref): preconnect to respec.org

### PR DESCRIPTION
**Pull request by sidvishnoi**

### CODE REVIEW

File: src/core/caniuse.js [hunk 1]
```diff
@@ -43,6 +43,12 @@ const supportTitles = new Map([
   ["d", "Disabled by default (needs to enabled)."],
 ]);
 
+const link = createResourceHint({
```
Comment (marcoscaceres) [inline src/core/caniuse.js]: Might be good to only add this if there isn't one already. So, maybe query the head for an existing one?

File: src/core/xref.js [hunk 2]
```diff
@@ -26,6 +27,12 @@ const profiles = {
 const API_URL = "https://respec.org/xref";
 const CACHE_MAX_AGE = 86400000; // 24 hours
 
+const link = createResourceHint({
+  hint: "preconnect",
+  href: "https://respec.org",
```
Comment (marcoscaceres) [inline src/core/xref.js]: As above, with removeOnSave.

File: src/core/caniuse.js [hunk 3]
```diff
@@ -43,6 +43,12 @@ const supportTitles = new Map([
...
+  hint: "preconnect",
+  href: "https://respec.org",
```
[the remark this answers is not in the document]
Comment (sidvishnoi) [reply src/core/caniuse.js]: This is done by `opts.dontRemove` (default false) in `createResourceHint`

[the remark this answers is not in the document]
Comment (marcoscaceres) [reply src/core/caniuse.js]: ah, so clever whomever designed this ;)

[the remark this answers is not in the document]
Comment (marcoscaceres) [reply src/core/caniuse.js]: ... shame he has no memory of any apis he wrote...

[the remark this answers is not in the document]
Comment (sidvishnoi) [reply src/core/caniuse.js]: and typescript saved me by shouting, as I added `class` :)

**Pull request merged**